tests/driver_pulse_counter/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+FEATURES_REQUIRED = periph_gpio
```

Comment:
I don't think this will be required if you add this as a depenency of the pulse_counter module in `drivers/Makefile.dep`

Reply:
addressed